### ISSUE TITLE
Unable to pull match details (e.g. participants, roster)

### ISSUE THREAD

**Issue by tsekylekobie** [cbortz/vainglory-api-ruby#7]
My code looks something like this:
```
db = VaingloryAPI(api_key)
match_info = db.matches({"filter[playerIds]" => id, "page[limit]" => numMatches})
```
I'm looking for a full match response similar to the data that I get when I run the following curl command:
```
curl -g "https://api.dc01.gamelockerapp.com/shards/na/matches?filter[createdAt-start]=2017-04-20T13:00:30Z&sort=-createdAt" \
-H "Authorization: Bearer apikey"  \
-H "X-TITLE-ID: semc-vainglory" \
-H "Accept: application/vnd.api+json" | jq
```

Currently, my `match_info` from my code only returns the `data` array of the full match response and none of the `included` array. How can I get access to the `included` array?

**Comment by cbortz** [comment]
Hey @tsekylekobie I'll try to look into this today, if not then tomorrow for sure!

**Comment by cbortz** [comment]
@tsekylekobie I found the issue, but I don't have time to patch it today. In the meantime, you can do something like this to get your data:

```ruby
db = VaingloryAPI(api_key)
req = db.matches({"filter[playerIds]" => id, "page[limit]" => numMatches})
match_info = JSON.parse(req.raw.body, object_class: OpenStruct)
```
You should then be able to access the other top level items such as `included` and `links`, using `match_info.included`.

Please let me know if this doesn't work for you. I plan to create a PR for this soon.

**Comment by tsekylekobie** [comment]
Awesome, the data's exactly what I was looking for. Thanks for the help @cbortz !

**Comment by cbortz** [comment]
@tsekylekobie I've created a pull request #8 if you'd like to take a look.

**Comment by cbortz** [comment]
Issue should be resolved with [release v0.0.4](https://github.com/cbortz/vainglory-api-ruby/releases/tag/v0.0.4).
